Tests/Shared.Specs/BasicEquivalencySpecs.cs

```diff
@@ -644,7 +644,7 @@ public void
             // Assert
             //-----------------------------------------------------------------------------------------------------------
             act.Should().Throw<XunitException>().WithMessage(
-                @"Expected member Member1 to be*""different"" with a length of 9, but*"""" has a length of 0.*");
+                @"Expected member Member1 to be*""different"" with a length of 9, but*"""" has a length of 0*");
```

Comment:
🔧  Is the exact exception message important in _all_ those tests? If not, I would only verify it where it's important.